updater/bootstrap.js

```diff
@@ -290,17 +290,51 @@ async function entry(info) {
         updaterArgs.push('-p');
         updaterArgs.push(statusWindow.webContents.getOSProcessId().toString());
     }
+    
+    const updaterStartCommand = `start \"\"  \"${updaterPath}\" `  
+    
+    updaterArgs.push('&& echo update_started_fine_app_can_be_closed');  
 
     log.info(updaterArgs);
 
-    cp.spawn(`${updaterPath}`, updaterArgs, {
+    const update_spawned = cp.spawn(`${updaterStartCommand}`, updaterArgs, {
         cwd: info.tempDir,
-        detached: true,
-        stdio: 'ignore',
+        detached: false,
+        //stdio: 'ignore',
```

Comment:
Let's delete this rather than commenting it out.  If there's a reason to leave it commented out then please add a comment above explaining why/when we would want to uncomment this in the future.